conans/client/downloaders/download.py

```diff
@@ -1,11 +1,9 @@
-from conans.client.downloaders.cached_file_downloader import CachedFileDownloader
 from conans.client.downloaders.file_downloader import FileDownloader
 
 
-def run_downloader(requester, verify, retry, retry_wait, download_cache,
-                   **kwargs):
+def run_downloader(requester, verify, retry, retry_wait, download_cache, **kwargs):
```

Comment:
Keeping the wrapper, but do we want to keep `download_cache` as well?

Reply:
Yes, you can introduce an assert to check it is never True